### ISSUE TITLE
how can you ensure that the output is a smooth sentence？

### ISSUE THREAD

**Issue by zhsamy** [macournoyer/neuralconvo#65]
I also refer to the paper Sequence to sequence learning with neural networks by tensorflow, but there are many repeat words in the outputs and the sentence is not smooth like yours. Would you mind telling me some tricks about your code?

**Comment by Conchylicultor** [comment]
Sorry if I'm late.
If the words are repeating, it may be because you don't train it long enough. It's usually the case at the beginning of the training. You can also try to change the learning rate see if it changes anything.

Otherwise, for the training tricks, here are some I discovered while doing a similar project: https://github.com/Conchylicultor/Deep-Learning-Tricks#seq2seq (see section seq2seq)
